client-v2/src/bundles/frontsUIBundle.ts

```diff
@@ -596,7 +594,7 @@ export {
   editorCloseAllOverviews,
   selectIsClipboardOpen,
   selectIsFrontOverviewOpen,
-  selectHasMultipleFrontsOpen
+  selectOpenFrontsCount
```

Comment:
this _was_ oddly specific

Reply:
I kinda liked it but either way - IT DID NOT SUIT MY NEEDS